### PR TITLE
[intro.scope] C++ is not a strict superset of C

### LINKED ISSUE #1664: intro.scope implies that C++ is a strict superset of C
In [intro.scope], the statement

> In addition to the facilities provided by C, C++ provides [ ... ]

implies that C++ is a strict superset of C.

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #1664.

### CODE REVIEW

**Comment by jensmaurer** [comment]
This change is a bit lame, but minimal.

File: source/intro.tex [hunk 1]
```diff
@@ -14,7 +14,7 @@
 \Cpp{} is a general purpose programming language based on the C
 programming language as described in ISO/IEC 9899:2011
 \doccite{Programming languages --- C} (hereinafter referred to as the
-\defnx{C standard}{C!standard}). In addition to
+\defnx{C standard}{C!standard}). In addition to some of
```
Comment (zygoloid) [inline source/intro.tex]: Hmm, how about:

> C++ provides many facilities beyond those provided by C, including [...]

This simultaneously minimizes the suggestion that C++ is a strict superset of C, and removes the suggestion that the following list of C++ features is a complete list of features in C++ but not C.

Comment (jensmaurer) [reply source/intro.tex]: Done.

**Comment by jensmaurer** [comment]
Rebased + implemented @zygoloid suggestion. (I fail to get excited about [intro.scope].)

**Pull request merged**